contracts/package.json

```diff
@@ -13,16 +13,12 @@
     "Solidity",
     "Hardhat"
   ],
-  "author": "",
   "license": "MIT",
   "devDependencies": {
     "@nomicfoundation/hardhat-toolbox": "^2.0.2",
-    "@nomiclabs/hardhat-web3": "^2.0.0",
-    "@openzeppelin/contracts": "^4.9.2",
-    "@openzeppelin/contracts-upgradeable": "^4.8.3",
-    "@openzeppelin/hardhat-upgrades": "^1.18.1",
-    "hardhat": "^2.17.0",
```

Comment:
You may have deleted `"hardhat": "^2.17.0"` by mistake, `@nomicfoundation/hardhat-toolbox` does not contain hardhat, the following is the package.json generated with `npx hardhat`:
``` json
{
   "name": "hardhat-project",
   "devDependencies": {
     "@nomicfoundation/hardhat-toolbox": "^3.0.0",
     "hardhat": "^2.17.1"
   }
}
```

Reply:
https://github.com/godwokenrises/godwoken-tests/pull/291/commits/3ccc6761848d01ceb238d3e7078f73c31be23501